sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -43,6 +57,83 @@ export type AnalyzeActionParameters<ActionName extends AnalyzeActionName> = {
     LanguageDetection: LanguageDetectionAction;
 }[ActionName];
 
+// @public
+export type AnalyzeBatchAction = {
+    actionName?: string;
+} & ((EntityLinkingAction & {
+    kind: "EntityLinking";
+}) | (EntityRecognitionAction & {
+    kind: "EntityRecognition";
+}) | (KeyPhraseExtractionAction & {
+    kind: "KeyPhraseExtraction";
+}) | (PiiEntityRecognitionAction & {
+    kind: "PiiEntityRecognition";
+}) | (SentimentAnalysisAction & {
+    kind: "SentimentAnalysis";
+}) | (HealthcareAction & {
+    kind: "Healthcare";
+}) | (CustomEntityRecognitionAction & {
+    kind: "CustomEntityRecognition";
+}) | (CustomSingleLabelClassificationAction & {
+    kind: "CustomSingleLabelClassification";
+}) | (CustomMultiLabelClassificationAction & {
+    kind: "CustomMultiLabelClassification";
+}));
+
+// @public
+export const AnalyzeBatchActionNames: {
+    readonly SentimentAnalysis: "SentimentAnalysis";
+    readonly EntityRecognition: "EntityRecognition";
+    readonly PiiEntityRecognition: "PiiEntityRecognition";
+    readonly KeyPhraseExtraction: "KeyPhraseExtraction";
+    readonly EntityLinking: "EntityLinking";
+    readonly Healthcare: "Healthcare";
+    readonly CustomEntityRecognition: "CustomEntityRecognition";
+    readonly CustomSingleLabelClassification: "CustomSingleLabelClassification";
+    readonly CustomMultiLabelClassification: "CustomMultiLabelClassification";
+};
+
+// @public
+export interface AnalyzeBatchOperationMetadata {
+    actionFailedCount: number;
+    actionInProgressCount: number;
+    actionSucceededCount: number;
+    createdOn: Date;
+    displayName?: string;
+    expiresOn?: Date;
+    lastModifiedOn: Date;
+    operationId: string;
+    status: State;
```

Comment:
It's a bit akward that the property is called `status` but the type is `State` any chance they can have the same name?

Reply:
Good point! I renamed `State` to `OperationStatus`.